pkgs/development/beam-modules/lib.nix

```diff
@@ -0,0 +1,21 @@
+{ pkgs }:
+
+rec {
+
+  /* Uses generic-builder to evaluate provided drv containing OTP-version specific data.
+
+  drv: package containing version-specific args;
+  builder: generic builder for all Erlang versions;
+  gargs: arguments passed to the generic-builder, used mostly to customize dependencies;
+  args: arguments merged into version-specific args, used mostly to enable/disable high-level OTP
+    features, like ODBC or WX support;
+
+  Please note that "mkDerivation" defined here is the one called from R16.nix and similar files.
+  */
+  callErlang = drv: gargs: args: pkgs.callPackage drv (
```

Comment:
Using `makeOverridable` for the arguments instead here would be more flexible and also makes the top-level definitions a bit nicer.

https://github.com/LnL7/nixpkgs/commit/de6ccfe987db09d0b32cb9eb6458a1a9ece89f8d